contracts/token/ERC20.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
 
-import "../utils/FilAddress.sol";
+import {FilAddress} from "../utils/FilAddress.sol";
```

Comment:
Since we now have duplicate errors defined in different files, if we import the entire file, we get duplicate error errors from the compiler

Reply:
ah, i was wondering why the syntax change. i wonder if this would be resolved if you placed the error definitions inside the contract/library definitions?

not a big deal either way though

Reply:
Oooh thats a good idea - i'm happy to test out the change if you want, just let me know